packages/odyssey-react/src/components/Field/index.tsx

```diff
@@ -26,6 +26,11 @@ export interface SharedFieldTypes {
    */
   optionalLabel?: string;
 
+  /**
+   * Prefix to be read by SRs before announcing an error.
+   */
+  errorPrefix?: string;
```

Comment:
can we avoid this specific dedicated prop here? Why not allow `error` to be a `ReactNode` so that someone can provide a mix of visible text and `<ScreenReaderText />` as necessary. Composition FTW!

Reply:
Updated.